ddtrace/_trace/trace_handlers.py

```diff
@@ -632,8 +632,9 @@ def _on_botocore_patched_bedrock_api_call_exception(ctx, exc_info):
     span = ctx[ctx["call_key"]]
     span.set_exc_info(*exc_info)
     prompt = ctx["prompt"]
+    model_name = ctx["model_name"]
```

Comment:
should we use `.get` to be more resistent to key errors here?

Reply:
ctx is a ddtrace internal object which extract model_name from the bedrock request, so we shouldn't worry about this key being contained or not.